src/Administration/Resources/app/administration/src/module/sw-extension/component/sw-extension-card-base/index.js

```diff
@@ -381,6 +390,43 @@ export default {
             await this.installExtension();
         },
 
+        openPrivilegesReviewModal() {
+            this.permissionModalActionLabel = this.$tc(
```

Comment:
`$tc` is deprecated in newer versions of `vue-i18n`. You can use `$t`


```suggestion
            this.permissionModalActionLabel = this.$t(
```

Reply:
Updated :)